src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -108,7 +108,7 @@ class JesBackend extends Backend with LazyLogging {
   def taskOutputToJesOutput(taskOutput: TaskOutput, callGcsPath: String, scopedLookupFunction: ScopedLookupFunction, engineFunctions: JesEngineFunctions): Try[Option[JesOutput]] = {
     taskOutput.wdlType match {
       case WdlFileType =>
-        taskOutput.expression.evaluate(scopedLookupFunction, engineFunctions) match {
+        taskOutput.expression.evaluate(scopedLookupFunction, engineFunctions, interpolateStrings = true) match {
```

Comment:
Turns out we kinda talked about it and then forgot about it. I left that night saying on hipchat I had an idea what he issues was, but was out the next day and everything got merged in. I forgot by the following day.